webpay/pay/views.py

```diff
@@ -147,22 +147,33 @@ def configure_transaction(request, data=None):
                     .format(mcc=mcc, mnc=mnc))
 
     is_simulation = request.session.get('is_simulation', False)
-    was_configured, error_code = tasks.configure_transaction(request,
-                                                             mcc=mcc, mnc=mnc)
-    if not was_configured and not is_simulation:
-        if not error_code:
-            error_code = msg.TRANS_CONFIG_FAILED
-        log.error('Configuring transaction failed: {er}'.format(er=error_code))
-        return system_error(request, code=error_code)
+    pay_req = request.session.get('notes', {}).get('pay_request')
```

Comment:
I don't think you need to be so defensive about this session var. Since we'll always be accessing this through the [api](https://github.com/mozilla/webpay/blob/master/webpay/api/api.py#L31-L34) it will always call ``process_pay_req()`` first which constructs the session. When we [refactor the pay module](https://bugzilla.mozilla.org/show_bug.cgi?id=1130477) we can remove the standalone ``configure_transaction()`` view.